gridcomps/History/Sampler/MAPL_TrajectoryMod_smod.F90

```diff
@@ -632,6 +657,14 @@
                      call lgr%debug('%a %a',   'exist: true filename       :', trim(filename))
                      call get_ncfile_dimension(filename, tdim=num_times, key_time=this%index_name_x, _RC)
                      len = len + num_times
+                     jj2 = jj2 + num_times
+                     if (j==this%obsfile_Ts_index) then
+                        this%obs(k)%count_location_until_matching_file = jj2
+                        write(6,'(2x,a,2x,i10)') 'this%obs(k)%count_location_until_matching_file', this%obs(k)%count_location_until_matching_file
```

Comment:
Do you really want an unguarded output here?    I/O to the console can slow the code.   Is this just for debugging?